<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CmdLog</class>
 <widget class="QWidget" name="CmdLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CmdLog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPlainTextEdit" name="te_log"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Command</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_cmd"/>
     </item>
     <item>
      <widget class="QPushButton" name="b_exec">
       <property name="text">
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
